docs/source/User-guide/Annotate.rst

```diff
@@ -5,168 +5,161 @@ Annotate
 
 MapReader's ``Annotate`` subpackage is used to interactively annotate images (e.g. maps).
 
-This is done in three simple steps:
-
-1. :ref:`Create_file`
-2. :ref:`Annotate_images`
-3. :ref:`Save_annotations`
+.. _Annotate_images:
 
-.. _Create_file:
+Annotate your images
+----------------------
 
-Create an annotation tasks file
------------------------------------
+.. note:: Run these commands in a Jupyter notebook (or other IDE), ensuring you are in your `mr_py38` python environment.
 
-To set up your annotation tasks, you will need to create a separate ``annotation_tasks.yaml`` file.
-An example file which can be used as a template can be found in ``MapReader/worked_examples/``.
 
-.. todo:: Note that you can do this via text editor in windows or something like ??? in mac/linux
+To prepare your annotations, you must specify a number of parameters when initializing the Annotator class.
+We will use a 'rail_space' annotation task to demonstrate how to set up the annotator.
```

Comment:
if it's baked in else where as `rail_space`, just leave this, but otherwise change to `railspace`

Reply:
Will make a separate ticket for this to go through and check in the docs